psalm-baseline.xml

```diff
@@ -946,7 +959,15 @@
       <code>new TestContainer('Default', $manager)</code>
     </InvalidPropertyAssignmentValue>
   </file>
+  <file src="test/ReflectionPropertyTrait.php">
+    <UnusedMethodCall>
+      <code>setAccessible</code>
+    </UnusedMethodCall>
+  </file>
   <file src="test/Config/SessionConfigTest.php">
+    <UnusedMethodCall>
+      <code>setAccessible</code>
+    </UnusedMethodCall>
```

Comment:
This can be fixed in the test by removing the call to `ReflectionProperty|Method::setAccessible()` and updating the baseline